gossip/hub/handlers_test.go

```diff
@@ -1140,16 +1189,30 @@ func TestGetSourceKeys(t *testing.T) {
 		t.Fatalf("getSourceKeys()=_,%v; want _,nil", gotErr)
 	}
 
-	var got api.GetSourceKeysResponse
-	if err := json.Unmarshal(gotRsp.Body.Bytes(), &got); err != nil {
+	var rsp api.GetSourceKeysResponse
+	if err := json.Unmarshal(gotRsp.Body.Bytes(), &rsp); err != nil {
 		t.Fatalf("Failed to unmarshal json response: %s", gotRsp.Body.Bytes())
 	}
-	want := api.GetSourceKeysResponse{Entries: []*api.SourceKey{{ID: testSourceID, PubKey: testSourcePubKeyDER}}}
-	if !reflect.DeepEqual(got, want) {
+	if got, want := sourceSet(t, rsp.Entries), sourceSet(t, testSources); !reflect.DeepEqual(got, want) {
```

Comment:
Alternate  idea / FYI about other libraries.

[`cmp.Transformer`](https://godoc.org/github.com/google/go-cmp/cmp#example-Option--SortedSlice) looks like it's setup to do exactly this kind of comparison. 
I think `cmp` also supports comparison of `[]byte` without needing to resort to `json.Marshal`

However, what you have here is also easy to follow.  No need to change it.

Reply:
Looks like it might be interesting/useful, but I'll stick with the current comparison (if only to not add a new dependency).